spec/core_functions/color/scale/a98_rgb.hrx

```diff
@@ -0,0 +1,225 @@
+<===> red/max/input.scss
+@use 'sass:color';
+a {b: color.scale(color(a98-rgb 0.2 0.5 0.7), $red: 100%)}
+
+<===> red/max/output.css
+a {
+  b: color(a98-rgb 1 0.5 0.7);
+}
+
+<===>
+================================================================================
+<===> red/min/input.scss
+@use 'sass:color';
+a {b: color.scale(color(a98-rgb 0.2 0.5 0.7), $red: -100%)}
+
+<===> red/min/output.css
+a {
+  b: color(a98-rgb 0 0.5 0.7);
+}
+
+<===>
+================================================================================
+<===> red/zero/input.scss
+@use 'sass:color';
+a {b: color.scale(color(a98-rgb 0.2 0.5 0.7), $red: 0%)}
+
+<===> red/zero/output.css
+a {
+  b: color(a98-rgb 0.2 0.5 0.7);
+}
+
+<===>
+================================================================================
+<===> red/low/input.scss
+@use 'sass:color';
+a {b: color.scale(color(a98-rgb 0.2 0.5 0.7), $red: -33%)}
+
+<===> red/low/output.css
+a {
+  b: color(a98-rgb 0.134 0.5 0.7);
+}
+
+<===>
+================================================================================
+<===> red/high/input.scss
+@use 'sass:color';
+a {b: color.scale(color(a98-rgb 0.2 0.5 0.7), $red: 86%)}
+
+<===> red/high/output.css
+a {
+  b: color(a98-rgb 0.888 0.5 0.7);
```

Comment:
I see.. I thought `scale()` was a simple multiplication, but I get it now... It's moving to the point that is 86% of the way from the original value to the maximum value

Reply:
Exactly :slightly_smiling_face:. The idea is to approximate the colloquial idea of "10% lighter" (or whatever) without blowing out colors that are already fairly light.